src/curve.js

```diff
@@ -297,6 +297,17 @@ export function getCurve(curve) {
     return null;
   }
   if (typeof curve == 'string') {
+    // If the curve is a custom cubic-bezier curve
+    if (curve.indexOf('cubic-bezier') != -1) {
+      const floatExp = '(\\d*.+\\d*)';
```

Comment:
changed to `(\\d+\\.?\\d*|\\.\\d+)` because I don't want extra parentheses that will produce extra matches.